src/fmu/dataio/_model/fields.py

```diff
@@ -556,6 +557,14 @@ class Context(BaseModel):
     See :class:`enums.FMUContext`."""
 
 
+class CaseContext(Context):
+    stage: Literal[enums.FMUContext.case] = Field(alias="stage")
+
+
+class IterationContext(Context):
+    stage: Literal[enums.FMUContext.iteration] = Field(alias="stage")
+
+
```

Comment:
@tnatt @mferrera Do you know of a way to set constant values for the `context.stage` without having these separate classes?

Reply:
Good question 🙂  I don't think so, not without using a `model_validator` to set the default value .. but not really an improvement over this!